package.json

```diff
@@ -17,9 +17,10 @@
     "apply-fmt": "prettier --write ."
   },
   "dependencies": {
+    "@bitgo/account-lib": "2.17.0-rc.1",
```

Comment:
this can be deleted, we dont need account lib in the project if its not there already